tests/test_workon.py

```diff
@@ -32,4 +32,9 @@ def test_no_symlink(env1):
 def test_no_pew_workon_home(workon_home):
     with temp_environ():
         os.environ['WORKON_HOME'] += '/not_there'
-        assert 'does not exist' in invoke('in', 'doesnt_exist').err
+        assert 'does not exist' in invoke('workon', 'doesnt_exist').err
```

Comment:
Yeah I wanted the test name to match the command executed :)
And the {{workon}}-based tests seem to pass perfectly